climada/entity/exposures/test/test_base.py

```diff
@@ -89,6 +90,20 @@ def test__init__meta_type(self):
         self.assertEqual("meta must be a dictionary",
                       str(cm.exception))
 
+    def test__init__geometry_type(self):
+        """ Check if the column name of geometry of GDF was customized"""
+        with self.assertRaises(ValueError) as cm:
+            Exposures(geometry = 'myname')
+        self.assertEqual("Exposures is not able to handle customized 'geometry' column names.",
+                      str(cm.exception))
+
+    def test__init__mda_in_kwargs(self):
+        """ Check if mda is not present in Kwargs after being removed"""
+        litpop = LitPop(exponents=2)
+        self.assertEqual(litpop.exponents, 2)
+        LitPop(meta=dict(exponents=2))
+        self.assertEqual(litpop.exponents, 2)
```

Comment:
In line 104 you create a `LitPop` object but do not assign it to any variable, you just leave it for the garbage collector to be removed.\
This action should in theory have no effect at all with the exception of some Logging messages that are written out.\
In line 105 you are checking again whether the object created in line 102 has an `exponents` value of `2`. But that was already checked in line 103. (If it really has changed since than we are in deep trouble! :scream:)

Reply:
Hi @emanuel-schmid , that was the tricky test we saw in the meeting with you and Chahan. I asked @peanutfun  help on it before the meeting and to explain it to me, he had to do it first. Then, since I thought it was finished, I decided to commit it with my other tests. I suggest that we cancel it and when I will know how to do it I will. Next time I won't commit a test that I don't understand and that I didn't do, even though I specified that in the commit. It would have been probably a better idea to not commit that, but I didn't knew what to do with it else then commit it.

Reply:
@NicolasColombi No worries! Even making the effort of raising a PR and explaining the test already helps us 🙌 

@emanuel-schmid You are right, this is a bug in the test. Here's my suggestion to fix it:

```suggestion
        litpop = LitPop(meta=dict(exponents=3))
        self.assertEqual(litpop.exponents, 3)
```